### ISSUE TITLE
FATAL ERROR on Install: change language on first page will raise error 500

### ISSUE THREAD

**Issue by migoi** [YetiForceCompany/YetiForceCRM#5449]
#### Issue

call install/Install.php and change the language from english to anything. This will cause in logs:
~~~
PHP Fatal error:  Uncaught App\Exceptions\BadRequest: ERR_NOT_ALLOWED_VALUE in vendor/yetiforce/Purifier.php:313
~~~
caused by:
~~~
Stack trace:
#0 vendor/yetiforce/Request.php(133): App\Purifier::purifyByType('', 2)
#1 vendor/yetiforce/Request.php(280): App\Request->getByType('mode', 2)
#2 install/views/Index.php(76): App\Request->getMode()
#3 install/Install.php(30): Install_Index_View->preProcess(Object(App\Request))
~~~

**Comment by migoi** [comment]
Maybe fix (at least a workaround:

Change "public_html/install/tpl/resources/Index.js" from:
~~~js
...
    changeLanguage: function(e) {
        var target = $(e.currentTarget);
        jQuery('input[name="mode"]').val('');
        jQuery('form[name="step1"]').submit();
    },
...
~~~

to
~~~js
...
    changeLanguage: function(e) {
        var target = $(e.currentTarget);
        jQuery('input[name="mode"]').val('step1');
        jQuery('form[name="step1"]').submit();
    },
...
~~~

**Comment by mariuszkrzaczkowski** [comment]
Merged #5450
